taxonomy-editor-frontend/src/pages/go-to-project/index.tsx

```diff
@@ -8,34 +8,54 @@ import EditIcon from "@mui/icons-material/Edit";
 import useFetch from "../../components/useFetch";
 import { API_URL } from "../../constants";
 import { toSnakeCase, toTitleCase } from "../../utils";
+import type { ProjectsAPIResponse } from "../../backend-types/types";
 
-const GotoProject = ({ resetNavLinks }) => {
-  const {
-    data: incomingData,
-    isPending,
-    isError,
-    errorMessage,
-  } = useFetch(`${API_URL}projects?status=OPEN`);
-  const [projectData, setProjectData] = useState([]);
+type ProjectType = {
+  id: string;
+  projectName: string;
+  taxonomyName: string;
+  branchName: string;
+  description: string;
+  errors_count: number;
+};
+
+type Props = {
+  resetNavLinks: () => void;
+};
```

Comment:
`Props` seems to be too generic. A better name for resetting the navLinks would be helpful.

Reply:
Think is ok to name it `Props` as the information that the components receive is called props. In case we have multiple components in the same file, I'd agree that we shouldn't name it only `Props`. Renamed `resetNavLinks` to `clearNavBarLinks`